Update news fragments with makefile changes (#1818)
cocotb/cocotb: documentation/source/newsfragments/1063.change.rst

> Questa, Aldec Riviera-PRO and GHDL now starts and saves results.xml file in Makefile location, the same behavior used by other simulators. (:pr:`1598`) (:pr:`1599`)

Reviewer: ```suggestion
Mentor Questa, Aldec Riviera-PRO and GHDL are now being started in the directory containing the Makefile and also save :file:`results.xml` there, the same behavior used by other simulators. (:pr:`1598`) (:pr:`1599`)
```

> Mentor Questa, Aldec Riviera-PRO and GHDL are now being started in the directory containing the Makefile and also save :file:`results.xml` there, the same behavior used by other simulators. (:pr:`1598`) (:pr:`1599`)

Reviewer 2: ```suggestion
Mentor Questa, Aldec Riviera-PRO and GHDL are now started in the directory containing the Makefile and also save :file:`results.xml` there, bringing them in line with the behavior used by other simulators. (:pr:`1598`) (:pr:`1599`)
```
